src/Elmah.Io.ElasticSearch/Elmah.Io.ElasticSearch.nuspec

```
@@ -2,21 +2,23 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <version>$version$</version>
-    <authors>elmah.io,jhilden</authors>
+    <authors>elmah.io,jhilden,Asgeir</authors>
```

nyoice!

---

I'll put my name too
```suggestion
    <authors>elmah.io,jhilden,Asgeir,gui.stracini</authors>
```